modules/db/src/test/resources/integration/config/oracle-datasource-config.xml

```diff
@@ -9,7 +9,7 @@
 
     <spring:bean id="jdbcDataSource" class="org.enhydra.jdbc.standard.StandardDataSource" destroy-method="shutdown">
         <spring:property name="driverName" value="org.apache.derby.jdbc.EmbeddedDriver"/>
-        <spring:property name="url" value="jdbc:oracle:thin:pocholo/password@172.16.28.128:1521:XE"/>
+        <spring:property name="url" value="jdbc:oracle:thin:pocholo/password@172.16.20.66:1521:XE"/>
```

Comment:
Points to a VM I used for testing, I changed all the values to be the same, so next time I need to test I can just replace all the values with one command